File: Podoal/app/src/main/java/com/creator/podoal/util/ProgressUtil.java
package com.creator.podoal.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.creator.podoal.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2023/PriceHunter/app/src/main/java/com/creator/pricehunter/util/ProgressUtil.kt
package com.creator.pricehunter.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.pricehunter.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}